frontend/src/lib/k8s/cluster.ts

```diff
@@ -138,14 +143,17 @@ export function makeKubeObject<T extends KubeObjectInterface | KubeEvent>(
 
     static apiList<U extends KubeObject>(
       onList: (arg: U[]) => void,
-      onError?: (err: ApiError) => void
+      onError?: (err: ApiError) => void,
+      opts?: {
+        namespace?: string;
```

Comment:
Is this type correct? I'd think it should be a ApiListOptions, with `namespace?: string | string[];`